reactive/kubernetes_node_base.py

```diff
@@ -58,10 +58,7 @@ def install_cni_plugins():
     set_flag("kubernetes.cni-plugins.installed")
 
 
-KUBERNETES_CP = "kubernetes-master"  # wokeignore:rule=master
-
-
-@when_any(KUBERNETES_CP + ".snaps.installed", "kubernetes-worker.snaps.installed")
+@when_any("kubernetes-node.snaps.installed")
```

Comment:
:heart: for the flag consolidation.. small nit:

```suggestion
@when("kubernetes-node.snaps.installed")
```